qemu/tests/cfg/rh_kernel_update.cfg

```diff
@@ -25,6 +25,10 @@
     RHEL.7:
         kernel_deps_pkgs += " kmod hmaccalc"
         upgrade_pkgs += " qemu-guest-agent"
+    RHEL.8:
```

Comment:
How about use `! RHEL.5, RHEL.6, RHEL7:`  ?  
I am ok with both, since future is uncertain.  It is up to you.

Reply:
I'm not sure if future guests will use same params as RHEL.8. If they use different ones, I guess it's better to specify for each kind of guest. What do you think?

Reply:
ok~ I am ok with that.